code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -779,12 +797,13 @@ GLOBAL_LIST_EMPTY(PDAs)
 	tnote += "<i><b>&rarr; To [target_text]:</b></i><br>[signal.format_message()]<br>"
 	// Show it to ghosts
 	var/ghost_message = "<span class='name'>[owner] </span><span class='game say'>PDA Message</span> --> <span class='name'>[target_text]</span>: <span class='message'>[signal.format_message(TRUE)]</span>"
-	for(var/mob/M in GLOB.player_list)
-		if(isobserver(M) && M.client && (M.client.prefs.chat_toggles & CHAT_GHOSTPDA))
-			to_chat(M, "[FOLLOW_LINK(M, user)] [ghost_message]")
+	if(length(string_targets))
+		for(var/mob/M in GLOB.player_list)
```

Comment:
Should be `for(/mob/dead/observer/O in GLOB.player_list)` considering there is already an observer typecheck below (and remove said typecheck).